.github/workflows/check_pull_requests_for_deploy.yml

```
@@ -15,10 +15,10 @@ jobs:
     - uses: actions/checkout@v2
     
     # Install dependencies
-    - name: Set up Python 3.7
+    - name: Set up Python 3.10
       uses: actions/setup-python@v1
       with:
-        python-version: 3.7
+        python-version: 3.10
```

```suggestion
        python-version: '3.10'
```
this is a super annoying thing… but, python version are actually strings not versions (remember they can be like 3.12.4) so it needs quotes